### DISCUSSION TITLE
automatically select newly pasted element in Tile Inspector

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16756] [Feature requests]
This would be a game changer! When you copy and paste a tile element in tile inspector, it currently does not select the newly pasted element. In some cases, you could accidentally delete the surface or a ride track because you forget to locate the newly pasted element. I would greatly appreciate a change in the default TI behavior to select newly pasted elements automatically.

**Comment by Broxzier** [comment]
The tile inspector already selects the pasted element, but only when no other element is already selected. When there is, it keeps the current selection. Simply deselecting the current selection  or reselecting the tile before pasting will give you the behaviour you want.

**Reply by 733737** [reply]
you mean i need to select another tile, then select the same tile then paste?

**Reply by Broxzier** [reply]
No, just deselect the element if you have one selected. That can be done by either clicking on an empty space in the list or by reselecting the same tile again.

**Reply by 733737** [reply]
reselecting the same tile again is very hard when you have varying height terrain or when working at heights over 200ft, unless you meant incrementing the x or y axis which I don't know if that will be a suitable way to do it?
